src/coreComponents/fileIO/Table/TableData.hpp

```diff
@@ -84,9 +85,10 @@ class TableData2D
   using ColumnType = real64;
 
   /// Struct containing conversion informations
-  struct Conversion1D
+  struct TableConversionData
```

Comment:
I can propose `TableDataHolder`, as you prefer.

Reply:
y i think it's better